enterprise/cypher/acceptance-spec-suite/src/test/resources/blacklists/cost-compiled.txt

```diff
@@ -1,220 +1,3 @@
-Case should handle mixed number types
```

Comment:
Why do we suddenly have so many removals from the blacklists? (compared to amount off added stuff below).

Reply:
Look at the whole list. I basically removed all of the old names and generated all blacklist entries with the new format. Each test from the old list should be in the new one (except the ones where we fixed a bug on the tck side)